public/js/app.js

```diff
@@ -1,45 +1,100 @@
-const displaySongs = function() {
-  const nowplayingImg = document.getElementById("nowplayingImg"),
-    nowplayingTitle = document.getElementById("nowplayingTitle"),
-    nowplayingInfo = document.getElementById("nowplayingInfo"),
-    lastPlayedDiv = document.getElementById("lastplayed"),
-    nowPlayingEndpoint = "https://35.211.223.26/nowplaying",
-    lastPlayedEndpoint = "https://35.211.223.26/lastplayed";
+const nowplayingImg = document.querySelector(".now-playing-img"),
+  nowplayingTitle = document.querySelector(".now-playing-title"),
+  nowplayingInfo = document.querySelector(".now-playing-info"),
+  nowPlayingArtist = document.querySelector(".now-playing-artist"),
+  recentlyPlayedDiv = document.querySelector(".recently-played-container"),
+  toggle = document.querySelector("#toggle"),
+  nowPlayingEndpoint = "https://35.211.223.26/nowplaying",
+  recentlyPlayedEndpoint = "https://35.211.223.26/recentlyplayed";
 
+const renderNowPlaying = song => {
+  const image = song.images.filter(i => i.width === 640);
```

Comment:
Looking back at this, I think it's better to use `song.images.find(i => i.width === 640)` which will return a single image or null.

Reply:
copy that, I'll update now